docs/user/azure/install_upi.md

```diff
@@ -288,11 +288,15 @@ Create the deployment using the `az` client:
 
 ```sh
 export VHD_BLOB_URL=`az storage blob url --account-name ${CLUSTER_NAME}sa --account-key $ACCOUNT_KEY -c vhd -n "rhcos.vhd" -o tsv`
+export RELEASE_VERSION=`openshift-install coreos print-stream-json | jq -r '.architectures.x86_64."rhel-coreos-extensions"."azure-disk".release'`
```

Comment:
probably also good to call out that you specify this based on architecture: `architectures.x86_64` or `architectures.aarch64`.

Reply:
I was thinking about doing that as a separate PR to add `aarch64` support.

Reply:
I decided to just do it in this PR.